devops/scripts/benchmarks/html/scripts.js

```diff
@@ -148,7 +148,14 @@ function createChart(data, containerId, type) {
     const chartConfig = {
         type: type === 'time' ? 'line' : 'bar',
         data: type === 'time' ? {
-            datasets: Object.values(data.runs)
+            datasets: Object.values(data.runs).map(runData => ({
+                ...runData,
+                // For timeseries (historical results charts) use runName,
```

Comment:
why? can't we always use displayLabel?

Reply:
For the Historical Results it's beneficial to have just a run name as a legend label. It's cleaner as all series in a chart differ only with a run name, the display name is all the same.